package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backdraft-app",
-  "version": "1.1.0-beta.1",
+  "version": "1.2.1-beta.2",
```

Comment:
So I made a mistake when I was publishing versions and put 1.2.0 out (when it wasn't perfect), so the version I'm shooting to release at the end of this ticket is 1.2.1.

Reply:
👍